plugins/chat/lib/chat/user_notifications_extension.rb

```diff
@@ -33,7 +43,7 @@ def chat_summary(user, _ = nil)
           AND chat_messages.created_at > now() - interval '1 week'
           AND (uccm.last_read_message_id IS NULL OR uccm.last_read_message_id < chat_messages.id)
           AND (uccm.last_unread_mention_when_emailed_id IS NULL OR uccm.last_unread_mention_when_emailed_id < chat_messages.id)
-          AND (chat_mentions.target_id = #{user.id} OR chat_mentions.type = 'Chat::AllMention')
+          AND ((chat_mentions.type = 'Chat::UserMention' AND chat_mentions.target_id = #{user.id}) OR (chat_mentions.type = 'Chat::AllMention' AND chat_channels.allow_channel_wide_mentions = true) #{group_sql})
```

Comment:
That line is starting to be very long. Maybe it's time to have it on multiple lines? 🤔

Reply:
Would it be possible to use a `CASE chat_mentions.type WHEN` here? 🤔

Reply:
Had to tweak this a little to put the END on the same line. We want to only check group mentions if user has mentionable groups so still handled this outside of the query (like before). I think this is more readable, nice suggestion 👍